bench-vortex/src/bin/clickbench.rs

```diff
@@ -86,21 +86,9 @@ struct DataFusionCtx {
     emit_plan: bool,
 }
 
-struct DuckDBCtx {
-    duckdb_path: PathBuf,
-}
-
-impl DuckDBCtx {
-    pub fn duckdb_file(&self, format: Format) -> PathBuf {
-        let dir = format!("clickbench_partitioned/{}", format.name()).to_data_path();
-        std::fs::create_dir_all(&dir).vortex_expect("failed to create duckdb data dir");
-        dir.join("hits.db")
-    }
-}
-
 enum EngineCtx {
     DataFusion(DataFusionCtx),
-    DuckDB(DuckDBCtx),
+    DuckDB(ddb2::DuckDBCtx),
```

Comment:
The 2 suffix will go away once TPC-H is moved over too.

Reply:
dont forget tpc-ds 🙈